crypto/keyring/keyring.go

```diff
@@ -494,7 +494,7 @@ func (ks keystore) List() ([]*Record, error) {
 		return nil, err
 	}
 
-	var res []*Record
+	var res []*Record //nolint
```

Comment:
linter suggest preallocate`res` like `res := make([]*Record, len(keys))` that will cause some refactoring and fixing tests, do you think it is a good idea?

Reply:
Done